Internationalisation

MaDumont opened the pull request:
The internationalisation is not completely finish, but 90% of it is in English now and that's better then 0. The select button is not really beautiful, but I'm not really good with that so someone else can do it.

MaDumont committed:
Merge branch 'master' of github.com:MaisonLogicielLibre/codetributhon

MaDumont committed:
Merge branch 'master' of https://github.com/maisonlogiciellibre/codetributhon

MaDumont committed:
Merge branch 'master' of github.com:MaisonLogicielLibre/codetributhon

MaDumont committed:
Merge branch 'master' of github.com:MaisonLogicielLibre/codetributhon

willkoua committed:
amelioration du design de la version mobile de la page des events

MaDumont committed:
Add some setup, so the project can translate pages.

Specify that the web site is in english.

MaDumont committed:
Add some setup, so the project can translate pages.

Specify that the web site is in english.

MaDumont committed:
Add some setup, so the project can translate pages.

Specify that the web site is in english.

MaDumont committed:
Add some setup, so the project can translate pages.

Specify that the web site is in english.

MaDumont committed:
Start translation of homes. Not finish.

The translation is only in english.

There is still 3 string that I'm not able to make it translate
on the page. I don't know why, I'm still searching for answers.

MaDumont committed:
Start translation of homes. Not finish.

The translation is only in english.

There is still 3 string that I'm not able to make it translate
on the page. I don't know why, I'm still searching for answers.

MaDumont committed:
Start translation of homes. Not finish.

The translation is only in english.

There is still 3 string that I'm not able to make it translate
on the page. I don't know why, I'm still searching for answers.

MaDumont committed:
Start translation of homes. Not finish.

The translation is only in english.

There is still 3 string that I'm not able to make it translate
on the page. I don't know why, I'm still searching for answers.

MaDumont committed:
Merge branch 'dev' of github.com:MaisonLogicielLibre/codetributhon into dev

Conflicts:
	codetributhon/static/css/libs/social-icons.css

MaDumont committed:
Delete definitions.py file because it's useless

It was a mistake to add at the first place,
so we can delete it.

MaDumont committed:
Delete definitions.py file because it's useless

It was a mistake to add at the first place,
so we can delete it.

MaDumont committed:
Delete definitions.py file because it's useless

It was a mistake to add at the first place,
so we can delete it.

MaDumont committed:
Delete definitions.py file because it's useless

It was a mistake to add at the first place,
so we can delete it.

MaDumont committed:
Translation of home page is  done.

Succeed to translate the last 3 chuck that was not working.

MaDumont committed:
Translation of home page is  done.

Succeed to translate the last 3 chuck that was not working.

MaDumont committed:
Translation of home page is  done.

Succeed to translate the last 3 chuck that was not working.

MaDumont committed:
Translation of home page is  done.

Succeed to translate the last 3 chuck that was not working.

MaDumont committed:
Button for changing language working

We can change for french to english with a select drop
down. We still have to show in url the language and make the
select look better.

MaDumont committed:
Button for changing language working

We can change for french to english with a select drop
down. We still have to show in url the language and make the
select look better.

MaDumont committed:
Button for changing language working

We can change for french to english with a select drop
down. We still have to show in url the language and make the
select look better.

MaDumont committed:
Button for changing language working

We can change for french to english with a select drop
down. We still have to show in url the language and make the
select look better.

MaDumont committed:
Translation of events section

MaDumont committed:
Translation of events section

MaDumont committed:
Translation of events section

MaDumont committed:
Translation of events section

MaDumont committed:
Translation of contact and reward section

MaDumont committed:
Translation of contact and reward section

MaDumont committed:
Translation of contact and reward section

MaDumont committed:
Translation of projects section

MaDumont committed:
Translation of projects section

MaDumont committed:
Translation of projects section

MaDumont committed:
Translation of footer and header

MaDumont committed:
Translation of footer and header

MaDumont committed:
Translation of footer and header

MaDumont committed:
Add the language in the url

MaDumont committed:
Add the language in the url

MaDumont committed:
Add the language in the url

MaDumont committed:
Add the language in the url

MaDumont committed:
Change the positionning of the language selector

MaDumont committed:
Change the positionning of the language selector

MaDumont committed:
Change the positionning of the language selector

MaDumont committed:
Change the positionning of the language selector

MaDumont committed:
Merge branch 'dev' of github.com:MaisonLogicielLibre/codetributhon into dev

MaDumont committed:
Merge branch 'internationalization' of github.com:MaDumont/codetributhon into internationalization

Conflicts:
	events/templates/events/detail.html

MaDumont committed:
Correction of little error

MaDumont committed:
Correction of little error

willkoua commented:
le changement de langue ne fonctionnne pas chez moi. L'URL change mais, pas le texte. Est ce que tu as envoye les fichiers de traduction??

Il faut aussi que dans ta description tu pr/cise les modalit/s pour faire fonctionnner ton module

MaDumont commented:
C'est moi qui a oublie de dire comment le tester. Il faut que tu fasse la commande python3 manage.py compilemessages avant de lancer le server.

Ca va creer des fichiers .mo, qui sont les fichiers de traductions

willkoua commented:
c bon il fonctionne

willkoua commented:
par contre il faudra creer une issue pour rendre le bouton plus userfriendly

MaDumont committed:
Merge branch 'internationalization' of github.com:MaDumont/codetributhon into internationalization

Conflicts:
	codetributhon/locale/fr/LC_MESSAGES/django.po
	codetributhon/settings.py
	codetributhon/urls.py
	events/locale/en/LC_MESSAGES/django.po
	events/locale/fr/LC_MESSAGES/django.po
	events/templates/events/detail.html
	pages/templates/pages/home.html

MaDumont committed:
Remake .po to be link with the actual text.

With the merge, there was some changes in the texte and we add to
be sure that the text in .po was corresponding to the text in file.

MaDumont committed:
Merge branch 'MaDumont-internationalization' into dev

MaDumont committed:
Merge branch 'dev' of github.com:MaisonLogicielLibre/codetributhon into dev

Conflicts:
	codetributhon/templates/footer.html
	codetributhon/templates/header.html

MaDumont committed:
Finalisation of the translation text with text in code.
